its/src/test/java/com/sonar/it/jenkins/SonarPluginTest.java

```diff
@@ -122,7 +124,11 @@ public void setUp() {
 
   @After
   public void cleanup() {
+    if (!wsClient.projects().search(new SearchRequest().setProjects(Collections.singletonList(MVN_PROJECT_KEY))).getComponentsList().isEmpty()) {
```

Comment:
There's another interesting way to replace the resetDat() => it's to do a bulk delete of all projects.
See how the languages team has do it for instance here : https://github.com/SonarSource/sonar-dotnet/blob/98c0671a0b71ec36f1fc7a237aa51a7b6e5f57d7/its/src/test/java/com/sonar/it/shared/TestUtils.java#L187

Reply:
nice, I'll use that